lib/features/upload/presentation/model/upload_file_state.dart

```diff
@@ -18,30 +18,37 @@ class UploadFileState with EquatableMixin {
   final int uploadingProgress;
   final Attachment? attachment;
   final CancelToken? cancelToken;
+  final bool fromFileShared;
```

Comment:
can you explain a little bit?

Reply:
Before @manh did the inline image insert, always insert it at the end of the editor by calling the `moveCursorAtLastNode` function. So I added the variable `fromFileShared` to be able to insert the image inline at the cursor position. And only insert at the end when the image is shared from another application.